src/SimpleThings/EntityAudit/AuditReader.php

```diff
@@ -748,12 +748,24 @@ public function getEntityValues($className, $entity)
     {
         $metadata = $this->em->getClassMetadata($className);
         $fields = $metadata->getFieldNames();
+        $associationsFields = $metadata->getAssociationNames();
 
         $return = array();
         foreach ($fields AS $fieldName) {
             $return[$fieldName] = $metadata->getFieldValue($entity, $fieldName);
         }
 
+        foreach($associationsFields as $key => $association) {
+            $associationValues = $entity->{'get' . ucfirst($association)}();
+            $values = array();
+            if ($associationValues instanceof \Doctrine\Common\Collections\Collection) {
+                $values[] = count($associationValues) > 0 ? array_map(array($associationValues->first(), 'getId'), $associationValues->toArray()) : array();
```

Comment:
* `count($associationValues) > 0` can be simplified with `count($associationValues)`
* This line is way too long. You can break it on ternary steps
* Some comments of why you are doing that would be great IMHO, it's hard to understand, so to maintain.